content/zh-cn/blog/java/codeanalysis/metrics/2-指标收集器的指标采集流程.md

```diff
@@ -1,10 +1,510 @@
 ---
 title: "2-指标收集器的指标采集流程"
 linkTitle: "2-指标收集器的指标采集流程"
-date: 2023-04-28
+date: 2023-05-16
```

Comment:
It would be better not to change publish date. SEO engine will save the origin address.

Reply:
Sure.I will fix it